tests/devapps/MSI/HelperService/MSIHelperService/Controllers/GetEnvironmentVariablesController.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Mvc;
+using MSIHelperService.Helper;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace MSIHelperService.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    [SwaggerTag(description: "Gets All Environment Variables")]
+    public class GetEnvironmentVariablesController : ControllerBase
```

Comment:
Why we shouldn't protect this API and instead rely on obfuscating values ? 

https://github.com/AzureAD/microsoft-identity-web/wiki/web-apis